src/dotnet/App.Maui/WebView/MauiWebView.cs

```diff
@@ -131,7 +132,7 @@ public bool MarkDead()
 
     private void OnUnloaded(object? sender, EventArgs eventArgs)
     {
-#if !WINDOWS
+#if IOS
```

Comment:
I am sure that it's not safe for IOS neither.
Inside DisconnectHandler they do blocking await on main thread
https://github.com/dotnet/maui/blob/main/src/BlazorWebView/src/Maui/iOS/BlazorWebViewHandler.iOS.cs#L133
that causes deadlock when there are async disposable services which do not complete synchronously.

Reply:
Then we should drop this case - I'm going to test iOS